Readme.md

```diff
@@ -135,9 +135,12 @@ Makemedia system requirements:
 
 **The starting point for this scenario is an MP4 or MOV file containing both video and audio** - ensure that your video is in this format before continuing. You can download some free test content in this format from the [Blender Foundation](https://mango.blender.org/download/). The steps below will transform this input video into a format suitable for playback.
 
-Before you can do anything with the video, you will need to generate a content key. The sample project includes a command line application for this. Simply execute the following command in the project directory: *node GenerateKey.js*
+Before you can do anything with the video, you will need to generate a content key. The sample project includes two simple command line applications that allow generating either a totally random key or a random key that is based on your key seed and a randomly generated key ID:
 
+* *GenerateKey.js* - generates a random content key. To use it, execute the following command in the project directory: *node GenerateKey.js*
```

Comment:
I'd leave it like this atm. Simple script and name :) Will change description a bit.